### PR TITLE
KOGITO-5011 - Unknown Custom tasks in Designer makes Diagram Explorer empty

### PR DESCRIPTION
**Pull request by Josephblt**
If an item could not be found in the workiItemDefinitionRegistry a null exception was thrown interrupting the load process.

**JIRA**: [KOGITO-5011](https://issues.redhat.com/browse/KOGITO-5011)

**Referenced Pull Requests**:
* https://github.com/kiegroup/kie-wb-common/pull/3636

**VS Code**: [VSCode Extension](https://drive.google.com/file/d/1j1NVlRrXkcO9rE8o0E402RWC-p62i_ZJ/view?usp=sharing)

### CODE REVIEW

**Comment by Josephblt** [comment]
Jenkins run cdb
Jenkins run fdb

**Comment by domhanak** [comment]
@Josephblt could you please rerun the FBD? there are new tests on VSCode IT tests so they might be useful to see if they work.

Will try the changes.

**Comment by domhanak** [comment]
@Josephblt FYI the VSIX is generated here - https://github.com/kiegroup/kogito-editors-java/pull/36/checks?check_run_id=2546704266 -> on top right corner at Arifacts. I installed the extension and was able to see explorer when an unknown task was in diagram. Thanks!

**Review by domhanak — APPROVED** [review]
Unable to reproduce the issues with this fix. Thanks! Approving. Please rerun the FDB to see that new WID covergae in tooling works.

**Comment by domhanak** [comment]
cc @romartin @handreyrc

**Comment by romartin** [comment]
please @Josephblt  lemme know when tests are passing and this is ready to merge. Thanks!

**Comment by domhanak** [comment]
Just for the record the failure on `bpmn-read-only` tests in standalone editors seems to be unrelated. Locally it passed.

**Comment by Josephblt** [comment]
Jenkins retest this

**Comment by Josephblt** [comment]
Jenkins do fdb

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java [hunk 1]
```diff
@@ -0,0 +1,155 @@
...
+    @Test
+    public void testViewHandler() {
+        when(imageProxy.isLoaded()).thenReturn(true);
+        tested.viewHandler().accept(task,
+                                    view);
+        verify(imageProxy, times(1)).load(eq(WID_ICON_DATA_1));
+    }
+
+    @Test
+    public void testGetGlyph() {
```
Comment (domhanak) [inline kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java:138]: Same as on old stream, could we make this separate test cases?

**Review by romartin — APPROVED** [review]

File: kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java (hunk 1 above)
Comment (romartin) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java:138]: as on the old stream too, waiting for resolving thsi comment in some way for merging

Comment (Josephblt) [reply kie-wb-common-stunner/kie-wb-common-stunner-sets/kie-wb-common-stunner-bpmn/kie-wb-common-stunner-bpmn-client/src/test/java/org/kie/workbench/common/stunner/bpmn/client/shape/def/CustomTaskShapeDefTest.java]: Done. Thanks @domhanak.

**Pull request merged**